docs/FrontierDevelopments-CAPI-endpoints.md

```diff
@@ -416,6 +437,50 @@ provides access to information about the Cmdr's fleet carrier.
     1. `value`: Value of commodity
     1. `stolen`: Whether the commodity is flagged as stolen
     1. `locName`: Localized name of commodity, follows carrier owner's localization
+
+1. `orders`: Information on all buy and sell orders
+    1. `commodities`: Buy and sell orders for standard cargo commodities
+        1. `sales`: A list of sell orders, in the following format:
+            1. `name`: The commodity name
+            1. `stock`: The quantity for sale
+            1. `price`: The sell price per item
+            1. `blackmarket`: Boolean, `true` if this item is for sale on the black market
+        1. `purchases`: A list of buy orders, in the following format:
+            1. `name`: The commodity name
+            1. `total`: The total buy order
+            1. `outstanding`: The remaining outstanding to fulfil the buy order
+            1. `price`: The buy price per item
+    1. `onfootmicroresources`: Buy and sell orders for micro resources (on foot materials)
+        1. `sales`: A Dict containing sell orders, with the key being the sell order ID (str) and the contents being a Dict in the following format:
```

Comment:
This could use a note about how empty `sales` is a list, although it's an object when it has data.